ui/components/MesheryPerformance/index.js

```diff
@@ -0,0 +1,1039 @@
+//@ts-check

```

Comment:
@kayceeDev I have already made this file and this is merged, please have another look at that.